browser/src/control/Control.JSDialogBuilder.js

```diff
@@ -2144,7 +2144,9 @@ L.Control.JSDialogBuilder = L.Control.extend({
 
 			controls['button'] = button;
 			var span;
-			if (builder.options.noLabelsForUnoButtons !== true && data.text) {
+			if (data.noLabel)
```

Comment:
Makes sense, toolitems usually inherit label visibility rules from the container.
Alternative could be to use some separate property instead of `text` for just saying it has only a `tooltip`.